assets/src/edit-story/elements/sticker/stickers/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import { default as SampleSticker } from './sampleSticker';
+export default {
+  SampleSticker,
+};
```

Comment:
Thinking about packages & bundling, we should separate the sticker element definition from the stickers source.

So the list of stickers would eventually reside in a `@web-stories-wp/stickers` package or something, while the element definition would be in a `@web-stories-wp/element` package.

With this in mind, it would be good to already move all stickers to `edit-story/stickers/`